src/phrases/phrases-en.ts

```diff
@@ -550,6 +550,9 @@ const phrases = {
       },
     },
   },
+  keyPerformanceIndicatorForm: {
+    title: 'Key numbers',
```

Comment:
Usikker på denne. Kanskje 'key figures' er bedre, eller 'statistics'. Egentlig burde kanskje alt hett Statistics i stedet for Key performance indicator.

Reply:
Enig der, tenkte selv at kanskje det blir forlangt